config/database.yml

```diff
@@ -24,6 +24,7 @@ default: &default
 development:
   <<: *default
   database: my_crud_development
+  password: root
```

Comment:
I prefer to have my default database user with no pass or use my computer credentials.
This change implies that everyone in you project will nee to set `root` as password. Its ok for now since you will be working alone, just keep in mind that.